portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectFieldAPI.macro

```diff
@@ -0,0 +1,68 @@
+definition {
+
+	macro _curlObjectField {
+		var portalURL = JSONCompany.getPortalURL();
+
+		if (isSet(objectDefinitionId)) {
+			var api = "object-definitions/${objectDefinitionId}/object-fields";
+		}
+		else {
+			var api = "/object-fields/${objectFieldId}";
+		}
+
+		var curl = '''
+			${portalURL}/o/object-admin/v1.0/${api} \
+			-H accept: application/json \
+			-H Content-Type: application/json \
+			-u test@liferay.com:test \
+		''';
+
+		return "${curl}";
+	}
+
+	macro deleteObjectField {
+		var curl = ObjectFieldAPI._curlObjectField(objectFieldId = "${staticObjectFieldId}");
+
+		JSONCurlUtil.delete("${curl}");
+	}
+
+	macro getIdOfcreatedObjectField {
+		Variables.assertDefined(parameterList = "${objectDefinitionId},${dbType},${name}");
+
+		var curl = ObjectFieldAPI._curlObjectField(objectDefinitionId = "${objectDefinitionId}");
+
+		if (!(isSet(label))) {
+			var label = "${name}";
+		}
+
+		var body = '''
+			-d {
```

Comment:
I think this may be an SF, you can check it again.

Reply:
Good catch, updated.